src/pal/src/init/pal.cpp

```diff
@@ -653,8 +655,35 @@ BOOL
 PALAPI
 PAL_IsDebuggerPresent()
 {
-    // Always retun false for now.
+#ifdef __LINUX__
+    BOOL debugger_present = FALSE;
+    char buf[1024];
```

Comment:
The size of /proc/self/status is pretty close to 1k on my machine. Would it make sense to make the buffer size 2k to make this a bit future proof?